allennlp/models/ensemble/bidaf_ensemble.py

```diff
@@ -0,0 +1,155 @@
+from typing import Dict, List, Any, Tuple
+
+from overrides import overrides
+import torch
+
+from allennlp.common.checks import ConfigurationError
+from allennlp.models.ensemble.ensemble import Ensemble
+from allennlp.models.archival import load_archive
+from allennlp.models.model import Model
+from allennlp.models.reading_comprehension import BidirectionalAttentionFlow
+from allennlp.common import Params
+from allennlp.data import Vocabulary
+from allennlp.training.metrics import SquadEmAndF1
+
+@Model.register("bidaf-ensemble")
+class BidafEnsemble(Ensemble):
+    """
+    This class ensembles the output from multiple BiDAF models.
+
+    It combines results from the submodels by taking the option with the most votes and breaking ties with
+    the average confidence of the start and stop indices.
+    """
+
+    def __init__(self,
+                 submodels: List[BidirectionalAttentionFlow]) -> None:
+        super(BidafEnsemble, self).__init__(submodels)
```

Comment:
You can just do `super().__init__(submodels)` here.  Including the `BidafEnsemble, self` arguments isn't necessary, and just adds clutter (I didn't know this until recently, so many of our classes have it, but we should remove them and not add more).